Whiskey/en-US/about_Whiskey_DotNetTest_Task.help.txt

```diff
@@ -0,0 +1,71 @@
+TOPIC
+    about_Whiskey_DotNetTest_Task
+
+SUMMARY
+    Runs unit tests for a .NET Core project.
+
+DESCRIPTION
+    The `DotNetTest` tasks runs the `dotnet test` command to run unit tests for a .NET Core project. Pass a list of solution files or .NET Core project files to the `Path` property. If no files are provided to `Path`, then the .NET Core SDK will search for any solution or project files in the working directory. If the `dotnet test` command returns a non-zero exit code the build will fail.

```

Comment:
change `non-zero exit code the build will fail` with `non-zerio exit code (i.e. tests failed), the build will fail)`.